### PR TITLE
Support Puppet agent 4.0 from Debian strech packages

### PR DESCRIPTION
**Pull request by sathieu**

### CODE REVIEW

**Comment by sathieu** [comment]
@mmol, you're probably interrested.

File: manifests/params.pp [hunk 1]
```diff
@@ -260,7 +260,11 @@
   if $aio_package {
     $client_package = ['puppet-agent']
   } elsif ($::osfamily == 'Debian') {
-    $client_package = ['puppet-common','puppet']
+    if (versioncmp($::puppetversion, '4.0') => 0) {
```
Comment (domcleal) [inline manifests/params.pp]: `>= 0`

Comment (sathieu) [inline manifests/params.pp]: yep, I added the = just before commiting (bad idea). Now fixed, sorry.

**Comment by sathieu** [comment]
I'm preparing the puppet-master part (another PR).

**Comment by domcleal** [comment]
Before you get too far, it might be worth waiting for @mmoll who has this WIP branch: https://github.com/mmoll/puppet-puppet/commits/puppet4_debian

**Comment by mmoll** [comment]
Yes, I wanted to wait for 4.5.1 (or .2) to get into testing before opening a PR here, but I tested my branch also in -installer with 4.5.0 in a complete foreman install, so I think mine is more complete.

**Comment by sathieu** [comment]
Great! I didn't knew about this. Will test your branch too.

**Pull request closed without merging**